src/components/ActionElement/ExtraActions.tsx

```diff
@@ -78,6 +79,7 @@ export function ExtraActions({
       </EuiFlexItem>
       <EuiFlexItem grow={false}>
         <SettingsPopover
+          isAssertion={actionContext.action.isAssert === true}
```

Comment:
Nit:

```suggestion
          isAssertion={actionContext.action.isAssert}
```

Reply:
This will break TS as `isAssert` is optional, but `isAssertion` requires a `boolean`. We could modify `isAssertion` to accept `undefined`, but that pushes the complexity into the implementation of `SettingsPopover`.

Alternatively, we could write something like `!!actionContext.action.isAssert`. I felt like the explicit check against `true` was fine, but I'm open to whatever you'd find most readable.

Reply:
Ah, that makes sense! Thanks for taking the time to explain. We can keep as is.